test/SlowTests/Client/Subscriptions/RevisionsSubscriptions.cs

```diff
@@ -533,10 +533,12 @@ public async Task CanReturnMetadataInRevisionsSubscription(string query)
                         }
                     };
 
-                    await Server.ServerStore.ModifyDatabaseRevisions(context,
+                    (long Index, object _) res = await Server.ServerStore.ModifyDatabaseRevisions(context,
```

Comment:
Shouldn't we check other test that use `ModifyDatabaseRevisions()` and also wait for the raft index?